### DISCUSSION TITLE
6.0.0 changelog?

### DISCUSSION THREAD

**Discussion by chrislgrigg** [ItsBenCodes/cookies#382] [Q&A] [answered]
Hello! I see that 6.0.0 and 6.0.1 were recently released. The 6.0.1 changelog is there but 6.0.0 is missing. Can you post it somewhere? I'm hoping it can help understand the upgrade process from 5.0.0 in a more succinct way than going through the git diff.

Thanks for your work!

**Comment by ItsBenCodes** [comment, marked as answer]
You should see it there: https://github.com/bendotcodes/cookies/releases/tag/v6.0.0
